Aztec.xcodeproj/xcshareddata/xcschemes/Aztec.xcscheme

```diff
@@ -56,6 +56,17 @@
                BlueprintName = "AztecTests"
                ReferencedContainer = "container:Aztec.xcodeproj">
             </BuildableReference>
+            <SkippedTests>
```

Comment:
These tests don't work under iOS 15, so they're disabled for now

Reply:
I was going to open a PR similar to https://github.com/wordpress-mobile/MediaEditor-iOS/pull/29 here, but I'll wait for input on my [question regarding the future of this library](https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1349#pullrequestreview-874770130).